shub/image/build.py

```diff
@@ -67,7 +67,7 @@ def build_cmd(target, version, skip_tests):
     click.echo("The image {} build is completed.".format(image_name))
     # Test the image content after building it
     if not skip_tests:
-        test_mod.test_cmd(target, version)
+        test_cmd.test_cmd(target, version)
```

Comment:
I'll merge this with gritted teeth, but my pedantic streak can't let it pass without commenting. ;)

A command is very narrowly defined thing specified by `click`. It must be an instance of `click.Command` (or at least behave like it). The `@click.command` decorator "creates a new `Command` and uses the decorated function as callback" . A module is a different very narrowly defined thing specified by the Python language.

Just because a module contains a `Command` at its heart (_among_ a bunch of helper functions) it doesn't suddenly become a command, and personally I find `test_cmd.test_cmd()` more difficult to digest than `test_mod.test_cmd()`. Why would a command live within another command of the same name?

```python
In [1]: from shub.image import test as test_cmd

In [2]: type(test_cmd)
Out[2]: module

In [3]: type(test_cmd.cli)
Out[3]: click.core.Command
```